File: PHP Excel Examples/CellsApiDemo/Column/unGroupColumns.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "UngroupColumns.xlsx";
$folder = "input";
$storage = null;
$sheetName = "Sheet4";
$firstIndex = 4;
$lastIndex = 9;
$apiInstance->ungroupColumns($name, $sheetName, $firstIndex, $lastIndex, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/column/ungroupColumns.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "UngroupColumns_1.xlsx"
sheetName = "Sheet4"
first_index = 4
last_index = 9
password = ""
storage = ""
folder = "/Cells/Column/"
api.ungroup_columns(name, sheet_name=sheetName, first_index=first_index, last_index=last_index, folder=folder, storage=storage)
